src/components/bbox/Bbox.tsx

```diff
@@ -69,6 +69,7 @@ interface IBboxProps {
   colorScheme?: IColorScheme;
   selectionMode?: TreeBboxSelectionMode;
   onClick?(e: any): void;
+  renderBbox?: (props: Omit<IBboxProps, 'renderBbox'>) => ReactElement;
```

Comment:
`renderBbox` should also be a property of the whole pdf viewer as is should be passed from the outside (e.g. from the the pdf4wcag-webapp-gui) and then passed to the Bbox. See the example of how it will be used:
```
import PdfViewer from '@duallab/verapdf-js-viewer';

...

<PdfViewer
    renderBbox={props => (
            <span
                style={{
                    position: 'absolute',
                    left: props.left + 'px',
                    top: props.top + 'px',
                    width: props.width + 'px',
                    height: props.height + 'px',
                    backgroundColor: 'blue',
                    opacity: 0.5,
                    color: 'transparent',
                }}
            />
        )}
    ...
/>

```